modules/react-components/src/components/modal/edit-avatar-modal/edit-avatar-modal.tsx

```diff
@@ -285,8 +285,8 @@ export const EditAvatarModal: FunctionComponent<EditAvatarModalPropsInterface> =
 
     useEffect(() => {
         if (imageUrl) {
```

Comment:
This `useEffect` is intended for a different purpose. Have you tested if the the behaviour is as expected with `showHostedURLOption` set to **true**.

Reply:
Also please verify the usage of this lifecycle and add a doc comment.

Reply:
Addressed with fc871b314e3cdff1f2ac4743b937e4e6f46ce825

Reply:
👍